services/web/src/helpers/modal.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Modal } from 'semantic';
+
+export default function modal(Component) {
+  class ModalWrapper extends React.Component {
+    state = {
+      open: this.props.open,
+      modelKey: Date.now(),
```

Comment:
I'm not really a fan of this method I think it's really unclear what's going on. We can discuss though... minimally I think you mean `modalKey` at least right?